src/main/kotlin/com/github/imflog/schema/registry/config/ConfigSubjectExtension.kt

```diff
@@ -0,0 +1,9 @@
+package com.github.imflog.schema.registry.config
+
+open class ConfigSubjectExtension {
+    val subjects: ArrayList<Pair<String, String>> = ArrayList()
+
+    fun subject(inputSubject: String, compatibility: String) {
```

Comment:
I think we should use the https://github.com/confluentinc/schema-registry/blob/master/client/src/main/java/io/confluent/kafka/schemaregistry/avro/AvroCompatibilityLevel.java
Like this we can use static typing in the object definitions even if in the end we have to use the `.name` of the enum in our rest call.

Reply:
Annoyingly, this isn't possible. The class has a public `name` field and a public method `name`, and Kotlin can't distinguish between them. What I will do is `valueOf()` the `String` to make sure it's an acceptable compatibility level before I pass it into the task.